public/app/features/dashboard/save_modal.ts

```diff
@@ -91,6 +91,9 @@ export class SaveDashboardModalCtrl {
   }
 
   compareTemplating() {
+    if (this.dashboardSrv.dash.originalTemplating.length !== this.dashboardSrv.dash.templating.list.length) {
+      return (this.variableChange = false);
```

Comment:
If the lists are not the same length, I would have assumed `variableChange` to be true. Unless `variableChange` has a different semantic.

Reply:
The variable change refers to changing variable while viewing and not adding and removing variables, I will add a comment that explains it in a good way.

Reply:
@davkal maybe changing name to variableValueChange would make it more clear